app/assets/stylesheets/application.css

```diff
@@ -126,3 +126,75 @@ body {
     background: #2ecc71;
 }
 
+.header-h1 {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 200px;
+    height: 200px;
+    margin: 30px auto;
+    border: 8px solid #2ecc71;
+    background: #fff;
+    border-radius: 30% 70% 70% 30%/30% 30% 70% 70%;
+    box-shadow: 15px 15px 50px rgba(0,0,0,0.2);
+    animation: anim-header-h1 10s infinite;
+    font-size: 24px;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+@keyframes anim-header-h1 {
+    0% {
+        border-radius: 30% 70% 70% 30%/30% 30% 70% 70%;
+        box-shadow: 15px 15px 50px rgba(0,0,0,0.2);
+    }
+    25% {
+        border-radius: 58% 42% 75% 25%/76% 46% 54% 24%;
+    }
+    50% {
+        border-radius: 50% 50% 33% 67%/55% 27% 73% 45%;
+        box-shadow: -10px -5px 50px rgba(0,0,0,0.2);
+    }
+    75% {
+        border-radius: 33% 67% 58% 42%/63% 68% 32% 37%;
+    }
+}
+.content{
+    display: inline-block;
+    margin-top: 20px;
+    width: 100%;
+}
+.content_head{
+    display:inline-block;
+    width: 100%;
+}
+.clientsbox {
+    background-color: #fff;
+    border: #2ecc71 solid;
+    margin: auto;
+    text-align: center;
+    vertical-align: middle;
+    position: relative;
+    width: 40%;
+    height: 60px;
+    display: inline-block;
+    align-items: center;
+    justify-content: center;
+    font-size: 12px;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+.clientscontainerHeader {
```

Comment:
The title of selectors need write with " _  " or " - ". Also u can use BEM methodolody.
![ex](https://user-images.githubusercontent.com/60444425/152578943-a088f813-74b8-4cbf-b9ad-8e56c6fa1e76.png)